resources/qml/Toolbar.qml

```diff
@@ -135,4 +156,15 @@ Item {
 
         visible: toolHint.text != "";
     }
+
+    UM.SettingPropertyProvider
+    {
+        id: machineExtruderCount
+
+        containerStackId: Cura.MachineManager.activeMachineId
+        key: "machine_extruder_count"
+        watchedProperties: [ "value" ]
+    }
+
+    UM.I18nCatalog { id: catalog; name: "cura" }
```

Comment:
However, in this case it turns out the catalog has become completely obsolete, so I removed it instead.